src/panels/config/automation/blueprint-automation-editor.ts

```diff
@@ -1,9 +1,7 @@
 import "@material/mwc-button/mwc-button";
 import { HassEntity } from "home-assistant-js-websocket";
-import { css, CSSResultGroup, html, LitElement } from "lit";
-import { customElement, property, state } from "lit/decorators";
-import { fireEvent } from "../../../common/dom/fire_event";
-import { nestedArrayMove } from "../../../common/util/array-move";
+import { html } from "lit";
+import { customElement, property } from "lit/decorators";
 import "../../../components/ha-alert";
 import "../../../components/ha-blueprint-picker";
```

Comment:
Remove unused imports

Reply:
After removing these, do I need to `import "blueprint-generic-editor"` to pick up these same imports from there, or is just importing the class sufficient?

Reply:
If you use the class not just for typing it is enough, if only used for typing it will be stripped out and thus not imported.